release-please-config.json

```diff
@@ -0,0 +1,39 @@
+{
+  "plugins": ["node-workspace"],
+  "bootstrap-sha": "78cc1e225bfc67300cfa9069fce257120fe92a6d",
+  "release-as": "21.4.0",
+  "packages": {
+    "packages/extension-api-explorer": {},
+    "packages/extension-sdk-react": {},
+    "packages/extension-sdk": {},
+    "packages/hackathon": {},
+    "packages/sdk-node": {},
+    "packages/sdk": {},
+    "packages/api-explorer": {
+      "release-as": ""
```

Comment:
this is how you override a global default `"release-as"` when you want the particular package to just auto-semver-increment based on conventional commit history.